docs/release-notes/10-4-0/README.md

```diff
@@ -68,7 +68,7 @@ In OpenProject you can now manage multiple building models in IFC format in para
 
 ![OpenProject-BIM-manage-IFC-models](OpenProject-BIM-manage-IFC-models.png)
 
-![OpenPoject-BIM_structural](OpenPoject-BIM_structural.png)
+![OpenProject-BIM_structural](OpenPoject-BIM_structural.png)
```

Comment:
Note: I did not fix the png file name - that is a reference to an actual file, which has a typo in the name.

Reply:
Done